scripts/zones/West_Ronfaure/mobs/Forest_Hare.lua

```diff
@@ -14,7 +14,7 @@ entity.onMobDeath = function(mob, player, isKiller)
 end
 
 entity.onMobDespawn = function(mob)
-    xi.mob.phOnDespawn(mob, ID.mob.JAGGEDY_EARED_JACK_PH, 5, math.random(3000, 21600)) -- 50 minutes to 6 hours
+    xi.mob.phOnDespawn(mob, ID.mob.JAGGEDY_EARED_JACK_PH, 9, 2400) -- 40 minute minimum
```

Comment:
Anecdotal, but I swear I've had JeJ pop back to back. I suspect he just has a stupidly low pop rate on retail rather than any cooldown. Since I can't prove it tho I'm not asking for a change. Just putting out there for any poor soul with too much time on their hands to look into.

Reply:
it is known that "true lottery" exists, wiki isn't an end all be all truth for things that can't really be known, like this. Without actual data taken accurately that minimum pop time may be nothing more than pure chance. I personally wouldn't trust anything but data from addons to remove the human element

Reply:
Is this holding up the PR? Not sure if you're wanting me to change what I have here for JEJ.

Reply:
no its not, need other reviewers

Reply:
Personally, most of the content of the zone audits is fine. However, there are certain details that would need maybe a wiki entry, a capture, a video or some kind of evidence.

Just my opinion though.

Reply:
I hadn't re-reviewed since mentioning the persistence module thing. I think did, 2 small bit you don't need, nothing else stands out to me.